specs/draft-cheshire-dnsext-dns-sd-02.txt

```diff
@@ -379,9 +379,9 @@ Internet Draft       DNS-Based Service Discovery      14th February 2004
    with the <Instance> portion, since it is allowed to contain any
    characters, including dots.
 
-   Any dots in the <Instance> portion should be escaped by preceeding
+   Any dots in the <Instance> portion should be escaped by preceding
```

Comment:
we can't go fixing any text in the 'specs' folder - these are external IETF specs for reference and need to be included verbatim.